### PR TITLE
Ensure subscription created_by Person and overwrite createsuperuser

### PR DESCRIPTION
**Pull request by faebebin**
#QF3203

https://app.clickup.com/t/2192114/QF-3203

Tested locally to create an organization and worked.

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3203 Fix field remaining_trial_organizations does not exist](https://app.clickup.com/t/2192114/QF-3203)

File: docker-app/qfieldcloud/settings.py [hunk 1]
```diff
@@ -96,6 +95,8 @@
     "auditlog",
     # Local
     "qfieldcloud.core",
+    # listed after core becuase we overwrite createsuperuser command
```
Comment (suricactus) [inline docker-app/qfieldcloud/settings.py:98]:
```suggestion
    # listed after core because we overwrite createsuperuser command
```

**Comment by suricactus** [comment]
I think the `createuser` command is still used by the k8s setup, we should not remove it, but fix it.

**Comment by faebebin** [comment]
Both `createuser.py` and `createsuperuser.py` create a Person instance now.

IMO we should nevertheless get rid of the createuser.py . Create followup task?

**Review by suricactus — APPROVED** [review]
Yes, I would prefer to have a single command too. I think it was mostly password related thing, it was robert197 who added this in the first place.

Thanks for the fixes!

**Pull request merged**